app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderRetryQueueItem.java

```diff
@@ -20,6 +20,7 @@
     @AttributeOverride(name = "tableName", column = @Column(name = "table_name", nullable = false, updatable = false, length = 100))
     @AttributeOverride(name = "operation", column = @Column(nullable = false, updatable = false, length = 1))
     @AttributeOverride(name = "snapshot", column = @Column(nullable = false, updatable = false))
+    @AttributeOverride(name = "requestUuid", column = @Column(name = "request_uuid", unique = true, updatable = false, length = 38))
```

Comment:
I don't think you should be adding this field, why are you adding it?

Reply:
When I was testing this ticket, I got the bellow error:
```
WARN o.h.e.j.s.SqlExceptionHelper:137 SQL Error: 1054, SQLState: 42S22
ERROR o.h.e.j.s.SqlExceptionHelper:142 Unknown column 'senderretr0_.requestUuid' in 'field list'
```

Then I looked at the table `sender_retry_queue`, and I found the column: `request_uuid  varchar(38)`, created by the bellow changeset:
```
    <changeSet author="wluyima" id="20211214-1601">
        <preConditions onFail="MARK_RAN">
            <not>
                <columnExists columnName="request_uuid" tableName="sender_retry_queue" />
            </not>
        </preConditions>
        <comment>Add request_uuid column to sender_retry_queue table</comment>

        <addColumn tableName="sender_retry_queue">
            <column name="request_uuid" type="VARCHAR(38)">
                <constraints unique="true" />
            </column>
        </addColumn>
    </changeSet>
```


So, to solve the error `Unknown column 'senderretr0_.requestUuid'` we should map the field to the correct column name, because de Embedded entity `org.openmrs.eip.component.entity.Event` does not have column mapping, leading to use the attribute name instead.

Am I missing something?

Reply:
Alright. I removed.

Reply:
But I don't seem to run into this issue? I wonder what you have on your side that I don't or vice versa

Reply:
Can you confirm if you have the request_uuid column in your sender management database?